src/ui/DynamicFacet/DynamicFacetRange.ts

```diff
@@ -87,6 +87,8 @@ export class DynamicFacetRange extends DynamicFacet implements IComponentBinding
      * The list of [range values]{@link IRangeValue} to request (see [Requesting Specific FacetRange Values](https://docs.coveo.com/en/2790/)).
      *
      * This value will override the [`numberOfValues`]{@link DynamicFacet.options.numberOfValues} value.
+     *
+     * If this option is not defined, the index will try to generate automatic ranges.
```

Comment:
@fbeaudoincoveo little piece of doc here 😃

Reply:
I'm ridiculously late to the party (I missed the PR review notification and only just saw today that a DOC Jira had been created for this), but for the record, I have no comments, the docs is good :)